docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,397 @@
+For example, the [`no-undef` rule](../../rules/no-undef) will create a violation for references to global variables not defined in the configured list of environments.
+
+[Flat configs](#flat-config) have globals defined as JavaScript objects.
+[Legacy configs](#legacy-config) have globals defined via predefined [environments](#environment).
+
+For information about configuring globals, see [Configure Language Options > Specifying Globals](../configure/language-options#specifying-globals).
+
+## I
+
+### Inline Config (Configuration Comment)
+
+A source code comment that configures a rule to a different severity and/or set of options.
+
+Inline configs use similar same syntax as [config files](#config-file-configuration-file) to specify any number of rules by name, their new severity, and optionally new options for the rules.
+For example, the following inline config comment simultaneously disables the `eqeqeq` rule and sets the `curly` rule to `"error"`:
+
+```js
+/* eslint eqeqeq: "off", curly: "error" */
+```
+
+For documentation on inline config comments, see [Rules > Using configuration comments](../configure/rules#using-configuration-comments).
+
+## L
+
+### Legacy Config
+
+The previous configuration file format for ESLint, now superseded by ["Flat" config](#flat-config).
+
+Legacy ELSint configurations are named in the format `.eslintrc.*` and allowed to be nested across files within sub-directories in a project.
+
+You can read more about the lifetime of legacy configurations in [ESLint's new config system, Part 1: Background](https://eslint.org/blog/2022/08/new-config-system-part-1).
+
+### Linter
+
+A [static analysis](#static-analysis) tool that can [report](#report) the results from running a set of [rules](#rule) on source code.
+Each rule may report any number of [violations](#violation) in the source code.
+
+ESLint is a commonly used linter for JavaScript and other web technologies.
+
+Note that a _linter_ is separate from [formatters](#formatter-tool) and [type checkers](#type-checker).
+
+### Logical (Rule)
+
+A [rule](#rule) that looks at logical issues in code.
+
+Many logical rules look for likely crashes (e.g. [`no-undef`](../../rules/no-undef)), unintended behavior (e.g. [`no-sparse-arrays`](../../rules/no-sparse-arrays)), and unused code (e.g [`no-unused-vars`](../../rules/no-unused-vars)),
+
+You can see the full list of logical rules that ship with ESLint under [Rules > Possible Problems](../../rules/#possible-problems)
+
+## N
+
+### Node
+
+A section of code within an [AST](#abstract-syntax-tree-ast).
+
+Each node has a node type representing its type of syntax.
+For example, the `1 + 2` in the AST for `1 + 2;` is a _BinaryExpression_.
+
+ESLint uses the [ESQuery](#esquery) format for its AST and [selectors](#selector).
+
+## O
+
+### Override
+
+When a [config file](#config-file-configuration-file) sets a new severity and/or rule options for a rule in a set of files.
```

Comment:
More accurate to say when a config object or configuration comment specifies a severity or options that supersede previously set severity or options.